massa-ledger-worker/src/ledger_db.rs

```diff
@@ -340,12 +343,14 @@ impl LedgerDB {
         // datastore
         let mut opt = ReadOptions::default();
         opt.set_iterate_upper_bound(end_prefix(data_prefix!(addr)).unwrap());
-        for (key, _) in self.db.iterator_cf_opt(
+        for res in self.db.iterator_cf_opt(
             handle,
             opt,
             IteratorMode::From(data_prefix!(addr), Direction::Forward),
         ) {
```

Comment:
You can use `.flatten()` and process only the `Ok`, idem line 390.
Or should we break if Err?

Reply:
done, thanks !